docs/SaveButton.md

```diff
@@ -0,0 +1,197 @@
+            ...
+        </SimpleForm>
+    </Edit>
+);
+```
+
+`<SaveButton>` accepts the following props:
+
+- [`icon`](#icon)
+- [`label`](#label)
+- [`mutationOptions`](#mutationoptions)
+- [`onClick`](#onClick)
+- [`sx`](#sx-css-api)
+- [`transform`](#transform)
+- [`type`](#type)
+
+Additional props (e.g. `color`, `variant`) are passed to [the underlying MUI `<Button>` component](https://mui.com/components/buttons/).
+
+## `icon`
+
+By default, `<SaveButton>` renders a disk icon. You can xcan pass another icon element:
+
+```jsx
+import AddBoxIcon from '@mui/icons-material/AddBox';
+import { SaveButton } from 'react-admin';
+
+const MySaveButton = props => <SaveButton {...props} icon={<AddBoxIcon />} />;
+```
+
+## `label`
+
+By default, `<SaveButton>` renders with the "Save" (translated if the user locale isn't English), whether used on a creation or edition form. You can pass another label:
+
+```jsx
+const PostCreateToolbar = () => (
+    <Toolbar>
+        <SaveButton label="Create post" />
+    </Toolbar>
+);
+```
+
+**Tip**: The label will go through [the `useTranslate` hook](./Translation.md), so you can use translation keys.
+
+## `mutationOptions`
+
+You can override the `mutationOptions` of the main mutation query (`dataProvider.create()` or `dataProvider.update()`) by passing a `mutationOptions` prop to `<SaveButton>`. This is useful when you have more than one save button.
+
+For instance, to display two save buttons in a creation form, one to save and redirect to the edition page, and the second to save and empty the form:
+
+{% raw %}
+```jsx
+import { Toolbar, SaveButton, useRedirect, useNotify } from 'react-admin';
+import { useFormContext } from 'react-hook-form';
+
+const MyToolbar = () => {
+    const { reset } = useFormContext();
+    const notify = useNotify();
+    return (
+        <Toolbar>
+            <SaveButton label="Save" />
+            <SaveButton 
+                label="Save and add"
+                mutationOptions={{
+                    onSuccess: () => {
+                        notify('Element created');
+                        reset();
+                    }}
+                }
+                type="button"
+                variant="text"
+            />
+        </Toolbar>
+    );
+};
+```
+{% endraw %}
+
+**Tip**: When using custom `mutationOptions`, you must set the button `type` to `button` instead of the default `submit`, otherwise it's the main page's `mutationOptions` that is used.
+
+## `onClick`
+
+You can add an event handler to the `<SaveButton>` by passing an `onClick` callback. 
+
+```jsx
+const PostCreateToolbar = () => (
+    <Toolbar>
+        <SaveButton label="Save" onClick={() => alert('Saving...')} />
+    </Toolbar>
+);
+```
+
+`onClick` it doesn't *replace* the default submission handler, but is instead called before it. To override the default submission handler, wrap a `<SaveButton>` in a custom [`SaveContext`](./useSaveContext.md).
```

Comment:
Might be worth mentioning that you can call `preventDefault` to cancel the submission